src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ApplicationTest.kt

```diff
@@ -243,26 +237,27 @@ class Cas2ApplicationTest : IntegrationTestBase() {
             unexpiredApplicationIds.add(createApplication(userEntity, offenderDetails).id)
           }
 
-          mockkStatic(OffsetDateTime::class)
-
-          unexpiredSubset.union(expiredSubset).forEachIndexed { index, status ->
+          unexpiredSubset.union(expiredSubset).forEachIndexed { _, status ->
```

Comment:
I think this can be simplified to forEach now instead of forEachIndexed? (there's a few other places this can be done)

Reply:
Yes - changed now.